### PR TITLE
Remove WebBundle

### PR DESCRIPTION
**Pull request by pamil**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | yes
| Related tickets | -
| License         | MIT

Not used anymore, but still annoying during some refactorings.

### CODE REVIEW

**Comment by loic425** [comment]
Is that normal that this PR has commits from #5648 ?

File: src/Sylius/Bundle/PayumBundle/Resources/views/layout.html.twig [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% extends 'SyliusWebBundle:Frontend:layout.html.twig' %}
+{% extends 'SyliusShopBundle::layout.html.twig' %}
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/PayumBundle/Resources/views/layout.html.twig]: We should not couple PayumBundle with ShopBundle

Comment (pamil) [inline src/Sylius/Bundle/PayumBundle/Resources/views/layout.html.twig]: :+1: - nevertheless I'm just moving the PayumBundle coupling from WebBundle to ShopBundle, it should be fixed in another PR.

**Comment by pjedrzejewski** [comment]
Thank you Kamil! :)
